website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -57,17 +57,52 @@ Use the Accordion when the toggle or the content is relatively complex.
 ![Accordion with complex content](/assets/components/accordion/usage-do-complex.png =770x*)
 !!!
 
-## Toggle content
+## Sizes
+
+The Accordion comes in three sizes: `small`, `medium`, and `large`. We recommend `medium` for most use cases, but use whichever size best fits your UI’s needs.
+
+![Example of various Accordion sizes](/assets/components/accordion/accordion-size-range.png)
+
+!!! Dont
+
+The size is set at the Accordion (group) level; don’t mix different sizes of Accordion items. This is a Figma-only issue.
+
+![Example of incorrectly combined Accordion sizes](/assets/components/accordion/accordion-size-Dont.png)
+!!!
+
+## Types
+
+The Accordion offers two container types: `card` and `flush` to fit different use cases.
+
+### Card
+
+The `card` variant is the default and recommended type. It creates a visual separation between accordion sections and other UI elements, organizes blocks of content, and distinguishes between sections within `tabs`.
+
+!!! Do
+
+![Example of appropriate usage of the Accordion card variant](/assets/components/accordion/accordion-card-type-example.png)
+!!!
+
+### Flush
+
+Use the `flush` variant where space is limited such as within a [Card](/components/card), [Flyout](/components/flyout), or Sidebar.
+
+!!! Do
+
+![Example of recommended usage of the Flush variant](/assets/components/accordion/accordion-flush-example.png)
+!!!
+
+## Toggle type
 
 The toggle accepts many different types of content, from text-based content to nested HDS components or custom content.
 
 ![Example of toggle content](/assets/components/accordion/toggle-content-text.png =690x*)
 
-### Generic content
+### Generic 
```

Comment:
I know we're updating these to "Generic", but for now, it should match what is in the actual component, which is "Custom"

```suggestion
#### Custom 
```